XboxKeyboardMouse/XboxController/Emulation/XboxStream.cs

```diff
@@ -88,20 +41,20 @@ public struct RECT {
                     Thread.Sleep(500);
 
                     IntPtr handle = GetForegroundWindow();
-                    if (GetWindowText(handle, text, count) > 0)
+
+                    var name = UWPUtils.GetProcessName(handle);
+
+                    if (!name.EndsWith("XboxApp.exe") && started)
```

Comment:
Sometimes, while switching between applications/windows, GetForegroundWindow can return a handle of 0.  GetProcessName then returns null, and the following `null.EndsWith` crashes the application.  We need null safety on name, and should probably ignore handle of 0 as well.